scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -85,6 +86,10 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
                 }
 
                 scope.isActive = !!item;
+
+                if (scope.config == null) {
+                    scope.isPackage = item != null && item.type === 'composite';
```

Comment:
We do we need two configs for this?
> `scope.isPackage = config != null && config.isPackage;`
> `scope.isPackage = item != null && item.type === 'composite';`

Reply:
Yes, there are two cases, the operation is for a single item or for a list of items